pkg/client/unversioned/request.go

```diff
@@ -48,6 +48,10 @@ import (
 // are therefore not allowed to set manually.
 var specialParams = sets.NewString("timeout")
 
+func init() {
+	metrics.Register()
```

Comment:
Yeah - the code before was correct - it wasn't correctness improvement.

It is just that we avoid some unnecessary checks (under lock) when creating every single request.